language-server/modules/langserver-core/src/test/resources/codeaction/convert-to-str-xml-tmplt/config/config8.json

```diff
@@ -0,0 +1,29 @@
+{
+  "position": {
+    "line": 28,
+    "character": 24
+  },
+  "source": "source.bal",
+  "expected": [
+    {
+      "title": "Convert to 'string' template",
```

Comment:
Can you recheck this test case? This should be a **negative** test case where we don't suggest the code action for constant expressions.

Reply:
yes. This should be.